epsagon/wrappers/aws_lambda.py

```diff
@@ -206,10 +221,11 @@ def _lambda_wrapper(*args, **kwargs):
             return result
         # pylint: disable=W0703
         except Exception as exception:
-            runner.set_exception(exception, traceback.format_exc())
+            runner.set_exception(exception, traceback.format_exc(), False)
             raise
         finally:
             try:
+                _add_status_code(runner, result)
```

Comment:
Step functions can be triggered by HTTP event?

Reply:
actually i think it can :)